src/pages/instances/InstanceIps.tsx

```diff
@@ -12,9 +12,13 @@ const InstanceIps: FC<Props> = ({ instance, family }) => {
   const addresses = getIpAddresses(instance, family);
   return addresses.length ? (
     <ExpandableList
-      items={addresses.map((address) => (
-        <div key={address} className="ip u-truncate" title={address}>
-          {address}
+      items={addresses.map((val) => (
+        <div
+          key={val.address}
+          className="ip u-truncate"
+          title={`${val.address}(${val.iface})`}
```

Comment:
Missing a space here:
```suggestion
          title={`${val.address} (${val.iface})`}
```